src/languages/en.js

```diff
@@ -680,6 +680,17 @@ export default {
             },
         },
     },
+    colorThemePage: {
+        colorTheme: 'Color theme',
```

Comment:
Let's match the mockups for copy i.e. `Color Theme` should be just `Theme`

<img width="530" alt="image" src="https://github.com/Expensify/App/assets/38015950/0310c1fc-2423-46a9-83db-f5dc3599980f">